### PR TITLE
Set Postfix compatibility level

### PR DESCRIPTION
**Pull request by detjensrobert**
As requested in osuosl-cookbooks/osl-mailrelay#4.

### CODE REVIEW

**Comment by detjensrobert** [comment]
[osl-mailrelay cookbook](https://trello.com/c/RcadQBNa/460-osl-mailrelay-cookbook)

File: attributes/default.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 default['osl-postfix']['main'] = {}
 default['osl-postfix']['main']['myorigin'] = '$mydomain'
 default['osl-postfix']['main']['smtpd_use_tls'] = 'no'
+default['osl-postfix']['main']['compatibility_level'] = '2'
```
Comment (ramereth) [inline attributes/default.rb:4]: I believe we only need this on EL8, can you please make that change?

**Review by ramereth — CHANGES_REQUESTED** [review]

**Review by ramereth — APPROVED** [review]
Patch bump please.

**Comment by detjensrobert** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.2.3.  Have a nice day!

**Pull request merged**